app/src/main/java/org/fossasia/openevent/app/common/utils/core/DateUtils.java

```diff
@@ -67,7 +68,14 @@ public static String formatDateToIso(@NonNull LocalDateTime date) {
     public static ZonedDateTime getDate(@NonNull String isoDateString) {
         if (isoDateString == null)
             return ZonedDateTime.now();
-        return ZonedDateTime.parse(isoDateString).withZoneSameInstant(getZoneId());
+        try {
+            exception = false;
+            return ZonedDateTime.parse(isoDateString).withZoneSameInstant(getZoneId());
+        } catch (DateTimeParseException pe) {
+            Timber.e(pe);
+            exception = true;
+            return ZonedDateTime.now();
+        }
```

Comment:
We never handle exceptions in util methods. If a string is invalid, why should we return current date to the user?